src/pytest_plugins/execute/eth_config/types.py

```diff
@@ -0,0 +1,147 @@
+                    activation_time=fork_activation_time,
+                    **network_kwargs,
+                )
+            else:
+                current_config = Config(
+                    fork=fork,
+                    activation_time=fork_activation_time,
+                    **network_kwargs,
+                )
+                break
+        assert current_config is not None, f"Unable to determine current fork at {current_time}"
+        for bpo_activation_time in reversed(self.blob_parameter_only_forks.keys()):
+            blob_schedule = self.blob_parameter_only_forks[bpo_activation_time]
+            if bpo_activation_time > current_time:
+                # The BPO activates at some point in the future.
+                if next_config is None:
+                    # The BPO is activated in the future and there's no other fork planned.
+                    # BPO is next.
+                    next_config = Config(
+                        fork=current_config.fork,
+                        activation_time=bpo_activation_time,
+                        **network_kwargs,
+                        blob_schedule=blob_schedule,
+                    )
+                else:
+                    # The BPO is acttivated in the future but there's already a fork (or other BPO)
+                    # scheduled.
+                    if bpo_activation_time > next_config.activation_time:
+                        # The BPO happens after the next scheduled fork, next fork happens first
+                        # so this BPO does not appear until next becomes current.
+                        pass
+                    elif bpo_activation_time == next_config.activation_time:
+                        # The BPO happens at the same time as the next scheduled fork,
+                        # so this blob configuration overrides the fork's one.
+                        next_config = Config(
+                            fork=next_config.fork,
+                            activation_time=bpo_activation_time,
+                            **network_kwargs,
+                            blob_schedule=blob_schedule,
+                        )
+                    else:
+                        # This BPO happens before the next fork, this BPO is next
+                        next_config = Config(
+                            fork=current_config.fork,
+                            activation_time=bpo_activation_time,
+                            **network_kwargs,
+                            blob_schedule=blob_schedule,
+                        )
+            else:
+                # The BPO is currently active
+                if bpo_activation_time >= current_config.activation_time:
+                    current_config.blob_schedule = blob_schedule
+                else:
+                    break
+
+        return (
+            current_config.get_config(),
+            next_config.get_config() if next_config is not None else None,
+        )
```

Comment:
Something like this to enable loading / saving network configs from / to JSON:
```suggestion
        )
        
  class NetworkConfigs(BaseModel):
      """Container for multiple network configurations."""

      networks: Dict[str, NetworkConfig]

      @classmethod
      def from_json_file(cls, file_path: str) -> "NetworkConfigs":
          """Load multiple network configs from JSON file."""
          with open(file_path) as f:
              return cls.model_validate_json(f.read())

      def to_json_file(self, file_path: str) -> None:
          """Save multiple network configs to JSON file."""
          with open(file_path, 'w') as f:
              f.write(self.model_dump_json(indent=2))

      def list_networks(self) -> List[str]:
          """List available network names."""
          return list(self.networks.keys())
```